substrate/frame/system/src/lib.rs

```diff
@@ -1032,6 +1030,12 @@ pub mod pallet {
 					})
 				}
 			}
+			#[cfg(feature = "experimental")]
+			if let Call::do_task { ref task } = call {
+				if task.is_valid() {
+					return Ok(ValidTransaction::default())
```

Comment:
```suggestion
					return Ok(ValidTransaction { priority: u64::max_value(), requires: Vec::new(), provides: vec![T::Hash::hash_of(&task)], longevity: TransactionLongevity::max_value(), propagate: false })
```

No need to propagate this transaction because this will created on every validator by the offchain worker. Or do you assume that someone sends this `do_task` manually?

Reply:
> Or do you assume that someone sends this do_task manually?

Yeah, they have an option to do as well, especially when the validator chooses to not run the off-chain worker?

Reply:
Not sure any one deactivates it :D But yeah, fine. Then only change `propagate` to `true`.

Reply:
(But the other stuff I added there is important) (maybe we should make the `longevity` smaller, otherwise the stay around in the state quite long.)

Reply:
Generally we should document that tasks can be executed in random order :D

Reply:
Yeah, sounds good. Thanks!